Sources/Extensions/UIKit/UIViewControllerExtensions.swift

```diff
@@ -96,5 +96,29 @@ public extension UIViewController {
         removeFromParent()
         view.removeFromSuperview()
     }
+
+    /// SwifterSwift: Helper method to present a UIViewController as a popover.
+    ///
+    /// - Parameters:
+    ///   - popoverContent: the view controller to add as a popover.
+    ///   - sourcePoint: the point in which to anchor the popover.
+    ///   - size: the size of the popover. Default uses the popover preferredContentSize.
+    ///   - delegate: the popover's presentationController delegate. Default is nil.
+    ///   - completion: The block to execute after the presentation finishes. Default is nil.
+    public func presentPopover(_ popoverContent: UIViewController, sourcePoint: CGPoint, size: CGSize? = nil, delegate: UIPopoverPresentationControllerDelegate? = nil, completion: ( () -> Void)? = nil) {
+
+        popoverContent.modalPresentationStyle = .popover
+
+        if let size = size {
+            popoverContent.preferredContentSize = size
+        }
+
+        let popoverPresentationVC = popoverContent.popoverPresentationController!
```

Comment:
Not sure, but this property can be nil on iPhones I think. 
As far as I remember popovers are iPad only

Reply:
Agreed. If anything, you might want to start the function with
```Swift
guard let popoverPresentationVC = popoverContent.popoverPresentationController else { return }
```
Also, you should almost never use `!`, it's a serious code smell in Swift.

Reply:
May I suggest using an `if let popoverPresentationVC = popoverContent.popoverPresentationController` ? 
This way this function will present a popover when on iPad or on iPhone when the delegate specifies so, and will not when the `presentationControllerVC` is `nil` (default situation on iPhone)